File: lib/result/ConnectionConfirmation.java
package com.teamlab.communication.wireless.lpr9201.result;

/**
 * リザルト ConnectionConfirmation
 */
public class ConnectionConfirmation extends Result {
	/**
	 * 
	 */
	public static final int RESULT_CODE = 0x00;

	/**
	 * 
	 */
	public static final int DATA_LENGTH_SIZE = 1; // byte

	/**
	 * 
	 * @param datas
	 */
	public ConnectionConfirmation(int[] datas) {
		super(RESULT_CODE, DATA_LENGTH_SIZE, datas);
	}
}

File: lib/result/connection_confirmation.js
/* jslint node: true */
"use strict"

/**
 * リザルト ConnectionConfirmation
 */

exports.ConnectionConfirmation = ConnectionConfirmation;

/**
 * 結果コード
 */
ConnectionConfirmation.RESULT_CODE = 0x00;

/**
 * データ長バイトサイズ
 */
ConnectionConfirmation.DATA_LENGTH_BYTE_SIZE = 1; // byte

/**
 * @param {Result} result
 */
function ConnectionConfirmation(result) {
  this.result = result;
}

/**
 * パース可能か
 * @param {Result} result
 */
ConnectionConfirmation.canParse = function(result) {
  return result !== null && result.resultCode === ConnectionConfirmation.RESULT_CODE;
}
